.claude/PINOKIO_LAUNCHER_GUIDE.md

```
@@ -0,0 +1,479 @@
+2. A Demo: the intent may be a 1-click launcher for the demo
+3. Open source project: the intent may be a 1-click launcher for the project 
+4. Regular website: the intent may be to clone the website and a launcher.
+5. There can be other cases, but try to guess.
+
+## Project Structure
+
+Pinokio projects normally follow a standardized structure with app logic separated from launcher scripts:
+
+Pinokio projects follow a standardized structure with app logic separated from launcher scripts:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)